examples/workflows/main.go

```diff
@@ -16,9 +16,9 @@ limitations under the License.
 
 // Example Allow List based access plugin.
 //
-// This plugin approves/denies access requests based on a simple Allow List
-// of usernames and associated allowed roles. Requests from allowed users
-// for allowed roles are approved, all others are denied.
+// This plugin atomatically approves access requests based on the approver's role.
```

Comment:
```suggestion
// This plugin automatically approves access requests based on the approver's role.
```

But it's also based on the allow list, no?